src/main/java/mate/academy/internetshop/dao/impl/ItemDaoImpl.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.internetshop.dao.impl;
+
+import java.util.NoSuchElementException;
+
+import mate.academy.internetshop.dao.ItemDao;
+import mate.academy.internetshop.dao.Storage;
+import mate.academy.internetshop.lib.Dao;
+import mate.academy.internetshop.model.Item;
+
+@Dao
+public class ItemDaoImpl implements ItemDao {
+    @Override
+    public Item add(Item item) {
+        Item any = Storage.items.stream()
```

Comment:
if you don't want to save the same items in the list, you can changed your list to set